packages/cli/src/lib/compare/compare-command.unit.test.ts

```diff
@@ -75,7 +74,8 @@ describe('compare-command', () => {
       { ...DEFAULT_CLI_CONFIGURATION, commands: [yargsCompareCommandObject()] },
     ).parseAsync();
 
-    expect(getLogMessages(ui().logger).at(-1)).toContain(
+    expect(ui()).toHaveLoggedLevel('info');
+    expect(ui()).toHaveLoggedMessage(
       `Reports diff written to ${bold(
```

Comment:
I would merge the level and message assertions into one. Apart from more convenient usage (IMHO), it will also ensure the same level-message pair is being checked.

```suggestion
    expect(ui()).toHaveLogged(
      'info',
      `Reports diff written to ${bold(
```